docgenerator/SDKDocGeneratorLib/output-files/unity-v3-migration.html

```diff
@@ -1,101 +0,0 @@
-<html>
```

Comment:
Ideally you should have a redirect or a placeholder if you remove a doc page.

Reply:
Ideally.

I will address all of these in one-go before final release.